### PR TITLE
fix: Extend date functions clobbering plus/minus

### PR DESCRIPTION
**Pull request by valya**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/workflow/src/Extensions/DateExtensions.ts [hunk 1]
```diff
@@ -161,7 +162,11 @@ function isWeekend(date: Date): boolean {
 	return [DAYS.saturday, DAYS.sunday].includes(DateTime.fromJSDate(date).weekday);
 }
 
-function minus(date: Date | DateTime, extraArgs: unknown[]): Date {
+function minus(date: Date | DateTime, extraArgs: unknown[]): Date | DateTime {
+	if (isDateTime(date) && extraArgs.length === 1) {
+		return date.plus(extraArgs[0] as DurationLike);
```
Comment (csuermann) [inline packages/workflow/src/Extensions/DateExtensions.ts:167]: shouldn't that be 'minus'? 🤔

Comment (valya) [reply packages/workflow/src/Extensions/DateExtensions.ts]: You are correct. I guess that's what I get for trying to quickly push out a fix on a Monday morning with no coffee :facepalm:

File: packages/workflow/src/Extensions/DateExtensions.ts [hunk 2]
```diff
@@ -161,7 +162,11 @@ function isWeekend(date: Date): boolean {
 	return [DAYS.saturday, DAYS.sunday].includes(DateTime.fromJSDate(date).weekday);
 }
 
-function minus(date: Date | DateTime, extraArgs: unknown[]): Date {
+function minus(date: Date | DateTime, extraArgs: unknown[]): Date | DateTime {
```
Comment (krynble) [inline packages/workflow/src/Extensions/DateExtensions.ts:165]: In order to remain consistent with the rest of the UI, shouldn't the functions always return primitives such as strings?

**Review by krynble — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.211.1`

**Pull request merged**